BUILD_OXYGEN.md

```diff
@@ -0,0 +1,51 @@
+
+   d. Click on the _OK_ button. The _Edit Parameter_ window will close.
+
+7. Now, you will be back in the _Edit Ant Scenario_ window again. If you are satisfied with your changes, click on the _OK_ button. Otherwise, you could also edit the directory where your schema gets stored by clicking on the _Output_ tab.
+
+8. Now, you will be back in your _Configure Transformation Scenario(s)_ window. In the _Projects_ or _Global_ section of the window based on your choice in step 5, you will find your _new project_ with the name you gave it in step 5. Click on it and then click on the _Applied associated_ button at the left-bottom corner of your window. This will build the schema.
+
+Once the building is done, Oxygen will automatically open the schema. The schema file is also stored in the `music-encoding/customizations/out/` folder if you want to consult it later. You can change the location where the schema generated is saved by clicking on _Output_ in the _Edit Ant Scenario_ window and changing the file path.
+
+#### Build Guidelines HTML
+
+In this section, we will use Oxygen to generate the HTML document for the guidelines using the XSLT Stylesheet located at `music-encoding/utils/guidelines_xslt/odd2html.xsl`. The steps are the following:
+
+1. Open the `mei-source.xml` file in Oxygen. This file is located in `music-encoding/source/mei-source.xml`.
+
+2. Click on the _Configure Transformation Scenario(s)_ button (the button with the wrench). This will open a window of the same name.
+
+3. Click on the _New_ button and select the _XML transformation with XSLT_ option. This will open the _New scenario_ window.
+
+4. Assign a name to your new XML transformation.
+
+5. In the _XSLT_ tab, look for the _XSL URL_ field and add the path to your `odd2html.xsl` file. You can do this by clicking on the folder icon to browse this file (it is located in your local copy of the music-encoding repo, in `music-encoding/utils/guidelines_xslt/odd2html.xsl`) and opening it.
+
+6. Now, you will be back in the _New Scenario_ window again. If you are satisfied with your changes, click on the _OK_ button. Otherwise, you could also edit the _XML URL_ field for a particular folder, as well as the _Parameters_ to change the output folder.
+
+7. Now, you will be back in your _Configure Transformation Scenario(s)_ window. In the _Global_ section of the window, you will find your new _XML transformation with XSLT_ with the name you gave it in step 4. Click on it and then click on the _Applied associated_ button at the left-bottom corner of your window. This will generate the HTML document for the guidelines.
+
+After a few minutes, the results of this build can be found in the web folder (`music-encoding/source/web`). The guidelines are stored in the `index.html` file.
```

Comment:
Shouldn't this go into the dist folder to be consistent with the ant build?

Reply:
thx for spotting just copy-pasted it, without re-reading

Reply:
Probably we should address this later and when doing so, rework the workflow description to use the `build.xml` and TEI Stylesheets of this repo instead of using the default shipping with oXygen.

Reply:
cf. #1117